Pwd segment

bug-ware opened the pull request:
The pwd-segment is similar to the path-segment but shows the folders as powerline using soft-separator. The option $PL_PATH_TRIM is handled the same as in path-segment.

chris-marsh reviewed (approved):
Nice.
1. Can you change the hard coded separator symbol to use  PL_SYMBOLS[soft_separator].
2. Remove the commented out ssh symbol reference on lines 5-7

bug-ware commented:
Hi. 

1. I hope I got it right. I think the hard coded symbol you mentioned was an ellipsis like symbol used to indicated the trimmed path? PL_SYMBOLS[soft_separator] was used already. I introduced segment symbols named pwd_separator and pwd_trimmed. 
2. I changed these lines to introduce pwd_* symbols. 

3. I was thinking to initialize pwd_separator with the value of `${PL_SYMBOL[soft_separator]}` instead of a hard coded symbol, i.e.:
`[[ -z ${PL_SYMBOLS[pwd_separator]} ]] && PL_SYMBOLS[pwd_separator]=${PL_SYMBOLS[soft_separator]}`
Will this give issues, e.g., if some changed soft_separator in the config after pwd_segment was inititalized? Seems to work though.
